doc/sal_constraints_and_recommendations.rst

```diff
@@ -509,9 +537,13 @@ Alternatively, upload the *testingest.sql* using the TBD webpage.
 
 `Proposed method of recording subsystem "Settings" data <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=58949768>`_
 
+.. _sal_constraints_and_recommendations:sal-topic-and-parameter-<description>-and-<unit>-fields:
```

Comment:
Please don't use < and > in section labels. Even if sphinx can handle it without complaining, it makes for needlessly awkward URLs.

If you are willing to change this, I suggest a global search for it -- I've found at least three instances.